src/XMLGenerator.php

```diff
@@ -499,6 +502,26 @@ public function updateUser(
         return $xml->asXML();
     }
 
+    /**
+     * Sanity-check's a User object before using it to generate XML for an
+     * updateUser request.
+     *
+     * The following checks are implemented:
+     *
+     * 1. If the User's SendEmailTo is set to Self, then the User's Email must
+     *     be set. Otherwise, a MissingValueException is thrown. See
+     *     https://support.smarteru.com/docs/api-updateuser#sendemailto-optional
```

Comment:
Not in scope of this PR but we ought to get back to auto-generating the docs on this project